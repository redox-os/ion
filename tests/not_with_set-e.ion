set -e

not true || echo not true
! true || echo not true
not false && echo not false
! false && echo not false
true && not false && echo true and not false
true && ! false && echo true and not false
not test -z "a" && echo not test
! test -z "a" && echo not test
not echo hello
echo $?
