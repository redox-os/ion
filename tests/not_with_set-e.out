not true
not true
not false
not false
true and not false
true and not false
not test
not test
hello
ion: pipeline execution error: early exit: pipeline failed with error code 1
